kuwala/scripts/initialize_windows.sh

```diff
@@ -1,12 +1,19 @@
 mkdir windows
 sed 's/\r$//' initialize_all_components.sh > ./windows/initialize_all_components.sh
 sed 's/\r$//' build_cli.sh > ./windows/build_cli.sh
+sed -i '1 a cd ..' ./windows/build_cli.sh
 sed 's/\r$//' build_jupyter_notebook.sh > ./windows/build_jupyter_notebook.sh
+sed -i '1 a cd ..' ./windows/build_jupyter_notebook.sh
 sed 's/\r$//' build_neo4j.sh > ./windows/build_neo4j.sh
+sed -i '1 a cd ..' ./windows/build_neo4j.sh
 sed 's/\r$//' create_zip_archive.sh > ./windows/create_zip_archive.sh
+sed -i '1 a cd ..' ./windows/create_zip_archive.sh
 sed 's/\r$//' initialize_core_components.sh > ./windows/initialize_core_components.sh
 sed 's/\r$//' initialize_git_submodules.sh > ./windows/initialize_git_submodules.sh
+sed -i '1 a cd ..' ./windows/initialize_git_submodules.sh
 sed 's/\r$//' run_cli.sh > ./windows/run_cli.sh
```

Comment:
The `run_cli.sh` also needs to have an additional change of directory.

Reply:
Yes, that's also the problem occurred when testing the branch... adding the fix now